var/spack/repos/builtin/packages/dbcsr/package.py

```diff
@@ -59,12 +63,15 @@ class Dbcsr(CMakePackage, CudaPackage, ROCmPackage):
             conflicts('+rocm', when='amdgpu_target={0}'.format(arch), msg=amd_msg)
 
     conflicts('+cuda', when='+rocm', msg="CUDA and ROCm are mutually exclusive")
-    conflicts('+rocm', when='@:2.0', msg="ROCm support is available in DBCSR v2.1 and later")
+    conflicts('+cuda', when='+opencl')
```

Comment:
Can you add a `msg` that `opencl` and `cuda` are mutually exclusive?

Reply:
Yes